packages/react-mutation-mapper/src/store/DefaultMutationMapperDataFetcher.ts

```diff
@@ -34,7 +38,11 @@ import {
 } from '../util/DataFetcherUtils';
 import { getMyVariantInfoAnnotationsFromIndexedVariantAnnotations } from '../util/VariantAnnotationUtils';
 import { uniqueGenomicLocations } from '../util/MutationUtils';
-import { getEvidenceQuery } from '../util/OncoKbUtils';
+import { OncoKbAnnotationQuery } from '../../../../src/shared/lib/StoreUtils';
+import {
+    AnnotateMutationByProteinChangeQuery,
+    AnnotateStructuralVariantQuery,
+} from 'cbioportal-frontend-commons/src/api/generated/OncoKbAPI';
```

Comment:
```suggestion
} from 'cbioportal-frontend-commons';
```

and then combine with the other imports from `cbioportal-frontend-commons` above